src/client/components/common/OptionList.tsx

```diff
@@ -45,7 +45,7 @@ const StyledSpinnerLoading = styled(SpinnerLoading)`
 
 const ArrowIcon = styled(Icon)<{ open?: boolean }>`
   margin-left: 0.8rem;
-  height: 1rem;
+  height: 0.8rem;
```

Comment:
<img width="185" alt="image" src="https://user-images.githubusercontent.com/78362532/161763248-4c7b255d-8e6b-476a-a598-5e6fab8c2fb1.png">
I'm seeing this on figma

Reply:
haha